etl/src/main/java/org/pih/hivmigration/etl/sql/ExamExtraMigrator.groovy

```diff
@@ -0,0 +1,64 @@
+package org.pih.hivmigration.etl.sql
+
+class ExamExtraMigrator extends SqlMigrator{
+
+    @Override
+    def void migrate() {
+
+        executeMysql("Create staging table for migrating next scheduled visit date", '''
+            create table hivmigration_exam_extra (                            
+              source_encounter_id int,
+              source_patient_id int,
+              next_exam_date date            
+            );
+        ''')
+
+        loadFromOracleToMySql('''
+            insert into hivmigration_exam_extra (
+              source_encounter_id,
+              source_patient_id,
+              next_exam_date 
+            )
+            values(?,?,?) 
+            ''', '''
+                select x.encounter_id as source_encounter_id,
+                        e.patient_id as source_patient_id,
+                        to_char(x.next_exam_date, 'yyyy-mm-dd') as next_exam_date
+                from hiv_exam_extra x,hiv_encounters e 
+                where x.next_exam_date is not null and x.encounter_id = e.encounter_id
+            ''')
+
+        executeMysql("Create tmp_obs_table", ''' 
+            CALL create_tmp_obs_table();
+            ''')
+
+        setAutoIncrement("tmp_obs", "(select max(obs_id)+1 from obs)")
+
+        executeMysql("Load next visit date as observations", ''' 
+                      
+            SET @next_visit_date_concept_uuid = '3ce94df0-26fe-102b-80cb-0017a47871b2';            
+                        
+            INSERT INTO tmp_obs (value_datetime, source_patient_id, source_encounter_id, concept_uuid)
+            SELECT next_exam_date, source_patient_id, source_encounter_id, @next_visit_date_concept_uuid
+            FROM hivmigration_exam_extra
+            WHERE next_exam_date is not null;
+            
+            CALL migrate_tmp_obs();
```

Comment:
I must have missed this one, but I assume this is a procedure that has been created to migrate tmp_obs into the obs table?

Reply:
@mogoodrich , this is a generic temp table and procedure that @brandones and @mseaton worked on while migrating the lab results:
https://github.com/PIH/hivmigration/commit/331175e5584015a640dc095b2fc90563aee7b9fd#diff-82cc5f8e97422fbfca840acb367e1bcfR3